addon/components/hyper-table.js

```diff
@@ -156,6 +157,22 @@ export default Component.extend({
     );
   },
 
+  groupByClusteringKey(fields) {
+    const map = new Map();
```

Comment:
just to know, why Map() instead of a basic `{}` ?

Reply:
+1